tests/integration/test_services_dedup_embedder_extractor_hasher_detector.py

```
@@ -0,0 +1,719 @@
+
+    def test_progress_callback_exception_does_not_abort(self, tmp_path: Path) -> None:
+        from file_organizer.services.deduplication.detector import (
+            DuplicateDetector,
+            ScanOptions,
+        )
+
+        content = "exception cb test"
+        (tmp_path / "ex1.txt").write_text(content)
+        (tmp_path / "ex2.txt").write_text(content)
+
+        def bad_cb(current: int, total: int) -> None:
```

Ruff (0.15.9)

[warning] Unused function argument: `current`

(ARG001)